src/main/webapp/components/workspaces/workspaces.js

```diff
@@ -154,13 +155,23 @@ export default () => {
   const workspacesSortedByTime = workspacesWithSubscriptions.sort(
     (a, b) => (a.attributes.modified > b.attributes.modified ? -1 : 1)
   )
+
+  const [filteredWorkspaces, setFilteredWorkspaces] = useState(
+    workspacesSortedByTime
+  )
```

Comment:
By maintaining the state of workspaces here we lose the ability to take advantage of the apollo cache's store of workspaces and inevitably we will run into syncing issues. Instead, I would recommend maintaining a state of a filter function that gets executed directly on the apollo cache store (i.e. the data object)